### ISSUE TITLE
Lightning Bolt spell damage scaling off

### ISSUE THREAD

**Issue by Felynda** [SunwellWoW/Sunwell-TBC-Bugtracker#1520]
Decription: 
Lightning bolt damage does not increase properly from spell damage.

How it works: 
In 40 casts i could not get 1 lightning bolt to hit has hard as should have been the minimum damage. 
Calculations for proof: my gear has 939 spelldamage. LB minimum damage naked is 585 i am specced concussion for 5% more dmg.

(939 spelldamage * 0.794 + 585) * 1.05 = 1397
So the minimum damage a LB should do is 1397. Currently my maximum hit cannot even surpass this as seen on the screenshot linked below:

https://imgur.com/a/L9Ytfdh

How it should work: 
Lightning bolt should have a 79.4% scaling of spell damage. LB has a higher then usual coeffecient because it used to be a 3 second cast spell till patch 2.3. Before 2.3 it was 3sec cast with -1sec from talent making it 2sec. 2.3 onward it became 2.5 sec spell with 2sec talented. To not nerf shaman dps this was added. 

Source (you should point out proofs of your report, please give us some source): 

Elitist Jerks article for proofs: http://web.archive.org/web/20081110100752/http://elitistjerks.com/f31/t16947-shaman_one_stop_elemental_shop/p11/

**Comment by notsofamous** [comment]
You need around 1000 casts to properly test a spell. 40 isn't a big enough pool of data.

**Comment by Felynda** [comment]
@notsofamous 40 is more then enough to show it. As said in first post my highest cast was much lower then the minimum should have been. My lightning bolts should hit between (939 spelldamage * 0.794 + 585) * 1.05 = **1397** and (939 spelldamage * 0.794 + 669) * 1.05 = 1485. As can be seen none of my bolts hit between this damage.

@Kokaina Character name was Felysha, as is also shown in the screenshot.

**Comment by zapsuu** [comment]
I tried with out any BSD and talents. I had 651 hit (splash LB is between 563-643 without talents), but I never saw a hit below 575.

I tried also rank 4 (splash 83-95). Hit was never below 88, the max hit was 99.

When I spend talents for Concussion (+5% dmg), the splash change:
Rank 4 (86-99) - min hit was 92, max 104.
Rank 12 (585-669)

I tried Rank12 + 235 BSD + concussion
0,789*235*1,05+585-669=780-864
My dmg: min 768, max 850 - 100 casts


Something definitly is wrong with coeffecient of this spell.
